src/components/BusinessList/BusinessList.tsx

```diff
@@ -4,12 +4,16 @@ import * as AR from "fp-ts/lib/Array";
 import { pipe } from "fp-ts/lib/function";
 import BusinessListItem from "../BusinessListItem/BusinessListItem";
 import { list, listItem } from "./BusinessList.treat";
+import ErrorView from "../../design-system/ErrorView/ErrorView";
 
 type Props = {
   businesses: Business[];
 };
 
 function BusinessList({ businesses }: Props) {
+  if (!businesses.length) {
+    return <ErrorView errorMessage="No element found :(" />;
+  }
   return (
     <ul className={list}>
       {pipe(
```

Comment:
Usually we try to avoid `if-else` and prefer ternary operator.
In this case even better (and you will see it quite often) something like this
`````
return nonEmptyArray.fromArray(business).
fold(
    <ErrorView/>,
    nonEmptyBusinessList => <ul>...</ul>
  )
`````
it may seems redundant the first times but you will see quite quickly how many magic tricks you can do with it.